packages/utilities/psammead-locales/moment/helpers/jalaali.js

```diff
@@ -0,0 +1,31 @@
+const moment = require('moment-timezone/moment-timezone');
```

Comment:
Does it not need to be this?
```suggestion
const moment = require('moment-timezone');
```

Reply:
This suggested change didn't fix the CI errors for me as `moment-timezone` was not available. I looked into it and I'm not sure why `moment-timezone` is even referenced, the line originates from https://github.com/bbc/moment-ws/pull/21 but none of the timezone API is used here AFAICS.

So instead in 265362b19ba8efce4c09a916b34facd41a76fc3d I've made `moment` a full dependency (since this package explicitly requires it) and removed `moment-timezone`. This seems correct to me but let me know if I'm missing something or you can see a better way. /cc @dr3